data/Templates/Hl7v2/ADT_A01.liquid

```diff
@@ -7,18 +7,23 @@
         {% if firstSegments.PID.3.1 -%}
             {% assign patientId = firstSegments.PID.3.1.Value | generate_uuid -%}
             {% assign fullPatientId = patientId | prepend: 'Patient/' -%}
+            {% include 'Resource/Patient' with PID: firstSegments.PID, PD1: firstSegments.PD1, NK1: firstSegments.NK1, ID: patientId -%}
```

Comment:
Is the previous `{% if patientId -%}` unnecessary any more?

Reply:
`{% if patientId -%}` equals to `{% if firstSegments.PID.3.1 -%}` in current logic.